wallets/provider-walletconnect-2/src/helpers.ts

```diff
@@ -47,58 +47,19 @@ type FinalNamespaces = {
   [key in NAMESPACES]?: ProposalTypes.BaseRequiredNamespace;
 };
 
-export function generateRequiredNamespace(
-  meta: BlockchainMeta[],
-  network: string
-): FinalNamespaces | undefined {
-  const evm = evmBlockchains(meta);
-  const cosmos = cosmosBlockchains(meta);
-
-  const requiredEvmChain = evm.find((chain) => chain.name === network);
-  const requiredCosmosChain = cosmos.find((chain) => chain.name === network);
-  const requiredSolanaChain = network === Networks.SOLANA;
-
-  if (requiredEvmChain) {
-    return {
-      [NAMESPACES.ETHEREUM]: {
-        events: DEFAULT_ETHEREUM_EVENTS,
-        methods: DEFAULT_ETHEREUM_METHODS,
-        chains: [
-          new ChainId({
-            namespace: NAMESPACES.ETHEREUM,
-            reference: String(parseInt(requiredEvmChain.chainId)),
-          }).toString(),
-        ],
-      },
-    };
-  } else if (!!requiredCosmosChain) {
-    return {
-      [NAMESPACES.COSMOS]: {
-        events: [],
-        methods: DEFAULT_COSMOS_METHODS,
-        chains: [
-          new ChainId({
-            namespace: NAMESPACES.COSMOS,
-            reference: requiredCosmosChain.chainId!,
-          }).toString(),
-        ],
-      },
-    };
-  } else if (requiredSolanaChain) {
-    return {
-      [NAMESPACES.SOLANA]: {
-        events: [],
-        methods: DEFAULT_SOLANA_METHODS,
-        chains: [`solana:${DEFAULT_SOLANA_CHAIN_ID}`],
-      },
-    };
-  }
-
-  return undefined;
+export function generateRequiredNamespace(): FinalNamespaces | undefined {
+  return {};
 }
 
+/*
+ * Since we didn't enable non-evm chains yet,
+ * it's better to not pass them in session proppsal request.
+ * As some wallets like 1inch wallet face with problem with passing non-evm chains
+ * even in optional name spaces.
+ */
 export function generateOptionalNamespace(
-  meta: BlockchainMeta[]
+  meta: BlockchainMeta[],
+  onlyEvm = true
```

Comment:
I couldn't find anywhere that passing `onlyEvm` as `false`. If we don't have any logic for that, we can remove these parts from this function and only keeps EVM.

Reply:
By the way, I could connect 1inch without facing any problem. is there any specific scenario you've encountered?

Reply:
Yes. if you pass optional cosmos / solana chain in namespace, it will fail.

Reply:
Thanks. much better. Could you please add this comment on top of `generateOptionalNamespace` as well? to ensure we don't add it in future or if we did, we should test android.